code/game/objects/explosion.dm

```diff
@@ -18,7 +18,7 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 		if(z_transfer && (devastation_range > 0 || heavy_impact_range > 0))
 			//transfer the explosion in both directions
 			explosion_z_transfer(epicenter, devastation_range, heavy_impact_range, light_impact_range, flash_range)
-///// Z-Level Stuff
+///// Z-Level Stuffis

```

Comment:
clearly a typo, cursor was probably in the wrong place when i typed something. i have no memory of this little change